kingpin/actors/rightscale/api.py

```diff
@@ -327,11 +345,11 @@ def update_server_array(self, array, params):
             <updated rightscale array object>
         """
 
-        log.debug('Patching ServerArray (%s) with new params: %s' %
-                  (array.soul['name'], params))
-        array.self.update(params=params)
-        updated_array = array.self.show()
-        return updated_array
+        log.debug('Patching %s with new params: %s' %
+                  (resource.soul['name'], params))
+        resource.self.update(params=params)
+        updated_resource = resource.self.show()
```

Comment:
Scratch that, I keep getting confused between the api and the actor. I see it's used in the actor